docs/tutorials/tutorial-extern.md

```diff
@@ -0,0 +1,206 @@
+     ```
+1. After Druid starts, open [http://localhost:8888/](http://localhost:8888/) in your browser to access the Web Console.
+1. From the [Query view](http://localhost:8888/unified-console.html#workbench), run the following command to load the Wikipedia example data set:
+     ```sql
+     REPLACE INTO "wikipedia" OVERWRITE ALL
+     WITH "ext" AS (
+       SELECT *
+       FROM TABLE(
+         EXTERN(
+           '{"type":"http","uris":["https://druid.apache.org/data/wikipedia.json.gz"]}',
+           '{"type":"json"}'
+         )
+       ) EXTEND ("isRobot" VARCHAR, "channel" VARCHAR, "timestamp" VARCHAR, "flags" VARCHAR, "isUnpatrolled" VARCHAR, "page" VARCHAR, "diffUrl" VARCHAR, "added" BIGINT, "comment" VARCHAR, "commentLength" BIGINT, "isNew" VARCHAR, "isMinor" VARCHAR, "delta" BIGINT, "isAnonymous" VARCHAR, "user" VARCHAR, "deltaBucket" BIGINT, "deleted" BIGINT, "namespace" VARCHAR, "cityName" VARCHAR, "countryName" VARCHAR, "regionIsoCode" VARCHAR, "metroCode" BIGINT, "countryIsoCode" VARCHAR, "regionName" VARCHAR)
+     )
+     SELECT
+       TIME_PARSE("timestamp") AS "__time",
+       "isRobot",
+       "channel",
+       "flags",
+       "isUnpatrolled",
+       "page",
+       "diffUrl",
+       "added",
+       "comment",
+       "commentLength",
+       "isNew",
+       "isMinor",
+       "delta",
+       "isAnonymous",
+       "user",
+       "deltaBucket",
+       "deleted",
+       "namespace",
+       "cityName",
+       "countryName",
+       "regionIsoCode",
+       "metroCode",
+       "countryIsoCode",
+       "regionName"
+     FROM "ext"
+     PARTITIONED BY DAY
+     ```
+
+### Query to export data
+
+Open a new tab and run the following query to export query results to the path:
+`/tmp/druid/wiki_example`.
+The path must be a subdirectory of the `druid.export.storage.baseDir`.
+
+
+```sql
+INSERT INTO
+  EXTERN(
+    local(exportPath => '/tmp/druid/wiki_example')
+        )
+AS CSV
+SELECT "channel",
+  SUM("delta") AS "changes"
+FROM "wikipedia"
+GROUP BY 1
+LIMIT 10
+```
+
+Druid exports the results of the query to the `/tmp/druid/wiki_example` directory.
+Run the following command to list the contents of 
+
+```bash
+ls /tmp/druid/wiki_example
+```
+
+The results are a CSV file export of the data and a directory.
+
+## Export query results to cloud storage
+
+The steps to export to cloud storage are similar to exporting to the local file system.
+Druid supports Amazon S3 or Google Cloud Storage (GCS) as cloud storage destinations.
+
+1. Enable the extension for your cloud storage destination. See [Loading core extensions](../configuration/extensions.md#loading-core-extensions).
+   - **Amazon S3**: `druid-s3-extensions`
+   - **GCS**: `google-extensions`
+  See [Loading core extensions](../configuration/extensions.md#loading-core-extensions) for more information.
+1. Configure the additional properties for your cloud storage destination. Replace `{CLOUD}` with `s3` or `google` accordingly:
```

Comment:
The numbering seems off here and in other places in the file.

Reply:
Using 1. for all the numbers lets the build system maintain the numbers for easier editing. Markdown translates the numbers to `<ol><li / ><li /><li /></ol>`

Reply:
Ah, got it! Thanks for explaining!